src/scroll-timeline-base.js

```diff
@@ -288,7 +288,9 @@ export class ScrollTimeline {
     let timeRange = calculateTimeRange(this);
 
     // Step 2
-    let currentScrollOffset = this.scrollSource.scrollTop;
+    let currentScrollOffset = this.orientation === 'block'
```

Comment:
Can you check for 'block' or 'vertical' and add a TODO similar to calculateScrollOffset (called above)?

Reply:
nice sure!

I've updated the PR to make inline scrolling the exception as well, felt more likely to be an exception than the rule. I tested and everything works great still.